hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleToken.java

```diff
@@ -91,59 +91,59 @@ public MerkleToken() {
 	}
 
 	public MerkleToken(
-			long expiry,
-			long totalSupply,
-			int decimals,
-			String symbol,
-			String name,
-			boolean accountsFrozenByDefault,
-			boolean accountKycGrantedByDefault,
-			EntityId treasury
+			final long expiry,
+			final long totalSupply,
+			final int decimals,
+			final String symbol,
+			final String name,
+			final boolean accountsFrozenByDefault,
+			final boolean accountKycGrantedByDefault,
+			final EntityId treasury
 	) {
 		this.expiry = expiry;
 		this.totalSupply = totalSupply;
 		this.decimals = decimals;
 		this.symbol = symbol;
 		this.name = name;
 		this.accountsFrozenByDefault = accountsFrozenByDefault;
-		this.accountsKycGrantedByDefault = accountKycGrantedByDefault;
+		accountsKycGrantedByDefault = accountKycGrantedByDefault;
```

Comment:
I think all the changes here, except for the `adminKey` getter, are not really necessary. It would be good to revert them.

Reply:
also, I think row 109 is pointless